iot/http_example/cloudiot_http_example.js

```diff
@@ -174,10 +200,14 @@ const devicePath = `projects/${argv.project_id}/locations/${argv.cloud_region}/r
 // The request path, set accordingly depending on the message type.
 const pathSuffix = argv.message_type === 'events'
     ? ':publishEvent' : ':setState';
-const url = `https://${argv.http_bridge_address}/v1beta1/${devicePath}${pathSuffix}`;
+const urlBase = `https://${argv.http_bridge_address}/v1beta1/${devicePath}`;
```

Comment:
Move this to the top of the file, or use a `let` and take advantage of hoisting? Seeing this declared after the functions that use it may confuse people who are new to JS (and don't understand that global statements get executed first).

Reply:
It uses the args, so at a minimum it needs to be after the argument parser... I have moved the variables up.